scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -328,6 +328,27 @@
     (catch Exception e
       (log/error e "Sandbox file server URL update error"))))
 
+(defn write-hostname-to-datomic
+  "Takes a hostname for an instance and saves it to datomic."
+  [conn task-id hostname] 
+  (log-structured/debug "Writing hostname" {:task-id task-id :hostname hostname})
+  (if task-id
+    (try
+      (let [db (db conn)
+            [instance] (first (q '[:find ?i
```

Comment:
I think this can be replaced by:

`(d/entity db [:instance/task-id task-id])`

See https://docs.datomic.com/on-prem/schema/identity.html#lookup-refs
and https://docs.datomic.com/on-prem/clojure/index.html#datomic.api/entity

Reply:
I will add this is a follow-up in addition to applying it to the above function for writing the sandbox URL.